### ISSUE TITLE
Linhazinha receber value types como props ao invés do objeto Todo

### ISSUE THREAD

**Issue by guiportooo** [vcavalcante/grupoReact#5]
Talvez seria legal o Componente **Linhazinha** receber as propriedades da Todo ao invés do objeto Todo. Isso traria algumas vantagens, como:

- O Componente **Linhazinha** pode ser agnóstico em relação a qual tipo de objeto será enviado, o que o torna mais reutilizável. Ou seja, qualquer outro Componente que queira exibir uma **Linhazinha** somente precisa fornecer as props _text_ e _checked_
- Caso o modelo de dados das Todos seja alterado, o Componente **Linhazinha** não precisará sofrer alterações
- Será possível utilizar _spread operator_ para enviar as props ao Componente **Linhazinha** :)

**Comment by chicocode** [comment]
Concordo com o @guiportooo mas por razões diferentes. Na minha opinião o que mais colabora para utilizarmos tipos primitivos é entender que o componente `Linhazinha` não deve conheçer nada sobre todo o contexto que vem com o objeto `todo`. 

O componente precisa somente saber sobre o texto, a função a ser chamada quando mudar o estado do checkbox e a propriedade `isComplete`, nada mais.

Isso deve simplificar o código, e ajudará a configurar melhor os `PropTypes` futuramente.
